framework/messages/hi/yii.php

```diff
@@ -0,0 +1,138 @@
+    'The file "{file}" is too small. Its size cannot be smaller than {formattedLimit}.' => 'फ़ाइल "{file}" बहुत छोटी है। इसका माप {formattedLimit} से छोटा नहीं हो सकता है।',
+    'The format of {attribute} is invalid.' => '{attribute} का प्रारूप अमान्य है।',
+    'The image "{file}" is too large. The height cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत बड़ी है। ऊंचाई {limit, number} {limit, plural, one{पिक्सेल} other {पिक्सेलस}} से बड़ी नहीं हो सकती।',
+    'The image "{file}" is too large. The width cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत बड़ी है। चौड़ाई {limit, number} {limit, plural, one{पिक्सेल} other{पिक्सेलस}} से बड़ी नहीं हो सकती।',
+    'The image "{file}" is too small. The height cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत छोटी है। ऊंचाई {limit, number} {limit, plural, one{पिक्सेल} other{पिक्सेल}} से छोटी नहीं हो सकती।',
+    'The image "{file}" is too small. The width cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत छोटी है। चौड़ाई {limit, number} {limit, plural, one{पिक्सेल} other{पिक्सेल}} से छोटी नहीं हो सकती।',
+    'The requested view "{name}" was not found.' => 'अनुरोधित दृश्य "{name}" नहीं मिला।',
+    'The verification code is incorrect.' => 'सत्यापन कोड गलत है।',
+    'Total <b>{count, number}</b> {count, plural, one{item} other{items}}.' => 'कुल <b>{count, number}</b> {count, plural, one{चीज} other{चीज़े}}।',
+    'Unable to verify your data submission.' => 'आपके डेटा सबमिशन को सत्यापित करने में असमर्थ।',
+    'Unknown alias: -{name}' => 'अज्ञात उपनाम: - {name}',
+    'Unknown option: --{name}' => 'अज्ञात विकल्प: - {name}',
+    'Update' => 'अद्यतन',
+    'View' => 'अवलोकन',
+    'Yes' => 'हाँ',
+    'Yii Framework' => 'Yii फ़्रेमवर्क',
+    'You are not allowed to perform this action.' => 'आपको यह कार्रवाई करने की अनुमति नहीं है।',
+    'You can upload at most {limit, number} {limit, plural, one{file} other{files}}.' => 'आप अधिकतम {limit, number} {limit, plural, one{फ़ाइल} other{फाइलें}} अपलोड कर सकते हैं।',
+    'in {delta, plural, =1{a day} other{# days}}' => '{delta, plural, =1{एक दिन} other{# दिनों}} में',
+    'in {delta, plural, =1{a minute} other{# minutes}}' => '{delta, plural, =1{एक मिनट} other{# मिनट}} में',
+    'in {delta, plural, =1{a month} other{# months}}' => '{delta, plural, =1{एक महीना} other{# महीने}} में',
+    'in {delta, plural, =1{a second} other{# seconds}}' => '{delta, plural, =1{एक सेकन्ड} other{# सेकन्डे}} में',
+    'in {delta, plural, =1{a year} other{# years}}' => '{delta, plural, =1{एक वर्ष} other{# वर्षों}} में',
+    'in {delta, plural, =1{an hour} other{# hours}}' => '{delta, plural, =1{एक घंटा} other{# घंटे}} में',
```

Comment:
'in {delta, plural, =1{an hour} other{# hours}}' => '{delta, plural, =1{एक घंटा} other{# घंटों}} में',